### PR TITLE
fixed test_ceph_s3.js script

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
1. fixed test_ceph_s3.js script that will run on docker.

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

**Review by nimrod-becker — APPROVED** [review]

File: src/test/system_tests/test_ceph_s3.js [hunk 1]
```diff
@@ -17,23 +17,28 @@ const promise_utils = require('../../util/promise_utils');
...
+        s3_sec_key = 'abc',
 } = argv;
 
 
 const api = require('../../api');
 let rpc = api.new_rpc();
 
 let client = rpc.new_client({
-    address: `ws://${mgmt_ip}:${mgmt_port}`
+    address: `${protocol}://${mgmt_ip}:${mgmt_port}` // http/https and give the mgmt ip and port
```
Comment (liranmauda) [inline src/test/system_tests/test_ceph_s3.js]: isn't the comment should be ws/wss?

File: src/test/system_tests/test_ceph_s3.js [hunk 2]
```diff
@@ -17,23 +17,28 @@ const promise_utils = require('../../util/promise_utils');
...
 });
 
 let auth_params = {
-    email: 'demo@noobaa.com',
-    password: 'DeMo1',
-    system: 'demo'
+    email: 'admin@noobaa.io',
+    password: `${pass}`,
+    system: 'noobaa'
```
Comment (liranmauda) [inline src/test/system_tests/test_ceph_s3.js]: When creating a local noobaa, isn't demo the default?

Comment (liranmauda) [reply src/test/system_tests/test_ceph_s3.js]: And we should also be able to pass the password via the src/test/framework/test_env_builder_kubernetes.js

Comment (romayalon) [reply src/test/system_tests/test_ceph_s3.js]: Running locally on mac results the following error: 's3-tests can only be run on Red Hat, Centos, Fedora, Ubunutu, or Debian platforms'.
I can keep it though but I believe that we won't run those tests locally again.
will talk to you on DM about the password issue.

File: src/test/system_tests/test_ceph_s3.js (hunk 1 above)
Comment (romayalon) [reply src/test/system_tests/test_ceph_s3.js]: ws gets socket hang up.. 
wss works fine..

**Pull request merged**